➡️ Gave some more details.

MilesWK opened the pull request:
First off, I like the name as is. 😃 

**On to my updates:**
* Made the `README.md` file *actually* give details. 👍 
* Changed the "title" element for the main page to "solder" 👍 

Hope you like the changes :)

— [MilesWK](https://www.mileswk.com/)

MilesWK committed:
Wow! I did so much!

acornitum commented:
feels chatgpt? going to close it since i already fixed the title before seeing this 🥲

MilesWK commented:
No! I didn't use chatGPT (look, for example, GPT comments all code) but
cool.

On Tue, Mar 25, 2025, 23:00 acon ***@***.***> wrote:

> feels chatgpt? going to close it since i already fixed the title before
> seeing this 🥲
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/solder/pull/1#issuecomment-2753119242>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A43POXWBJ5JMY26XKJEJNDD2WIJ3FAVCNFSM6AAAAABZTOGVGKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONJTGEYTSMRUGI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> [image: acornitum]*acornitum* left a comment (hackclub/solder#1)
> <https://github.com/hackclub/solder/pull/1#issuecomment-2753119242>
>
> feels chatgpt? going to close it since i already fixed the title before
> seeing this 🥲
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/solder/pull/1#issuecomment-2753119242>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A43POXWBJ5JMY26XKJEJNDD2WIJ3FAVCNFSM6AAAAABZTOGVGKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONJTGEYTSMRUGI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

acornitum commented:
alright, thanks for trying regardless!

not going to accept the commits since i dont think they add the value we need at this current stage.
